library/functions.po

```diff
@@ -403,221 +397,198 @@ msgid ""
 msgstr ""
 
 #: ../../library/functions.rst:136
-#, fuzzy
 msgid ""
 "Return a new array of bytes.  The :class:`bytearray` class is a mutable "
 "sequence of integers in the range 0 <= x < 256.  It has most of the usual "
 "methods of mutable sequences, described in :ref:`typesseq-mutable`, as well "
 "as most methods that the :class:`bytes` type has, see :ref:`bytes-methods`."
 msgstr ""
-"返回一個新的 bytes 陣列。 :class:`bytearray` 類是一個可變序列，包含範圍為 0 "
-"<= x < 256 的整數。它有可變序列大部分常見的方法，見 :ref:`typesseq-mutable` "
-"的描述；同時有 :class:`bytes` 型別的大部分方法，參見 :ref:`bytes-methods`。"
+"返回一個新的 bytes 陣列。 :class:`bytearray` 類別是一個可變的整數序列，包含範"
```

Comment:
There's a redundant space after the period.

Great catch on noticing the "sequence of *integers*" in the original message!